roles/config-packer/tasks/main.yaml

```diff
@@ -7,9 +7,6 @@
     - name: install packer from source
       shell: |
         set -ex
-        mkdir -p $GOPATH/src/github.com/hashicorp/packer
-        cp -r '{{ zuul.project.src_dir }}' $GOPATH/src/github.com/hashicorp/
-        cd $GOPATH/src/github.com/hashicorp/packer
```

Comment:
maybe need to add a `chdir: '{{ zuul.project.src_dir }}'` if we remove the `cd` statement ?

Reply:
Thank you, will update PR